Jenkinsfile-datastax

```diff
@@ -371,30 +343,10 @@ pipeline {
                           <td><strong>1.8</strong></td>
                           <td>Oracle JDK version 1.8 (<i>Used for compiling regardless of choice</i>)</td>
                         </tr>
-                        <tr>
-                          <td><strong>openjdk@1.9</strong></td>
-                          <td>OpenJDK version 9</td>
-                        </tr>
-                        <tr>
-                          <td><strong>openjdk@1.10</strong></td>
-                          <td>OpenJDK version 10</td>
-                        </tr>
                         <tr>
                           <td><strong>openjdk@1.11</strong></td>
                           <td>OpenJDK version 11</td>
                         </tr>
-                        <tr>
-                          <td><strong>openjdk@1.12</strong></td>
-                          <td>OpenJDK version 12</td>
-                        </tr>
-                        <tr>
-                          <td><strong>openjdk@1.13</strong></td>
-                          <td>OpenJDK version 13</td>
-                        </tr>
-                        <tr>
-                          <td><strong>openjdk@1.14</strong></td>
-                          <td>OpenJDK version 14</td>
-                        </tr>
```

Comment:
Non-LTS versions which are already EOL.  We also aren't aiming to explicitly support anything that isn't an LTS Java release so in the interest of keeping this simple I just yanked all of these.